network/p2p/p2p_sync.go

```diff
@@ -259,7 +259,9 @@ func (n *p2pNetwork) makeSyncRequest(logger *zap.Logger, peers []peer.ID, mid sp
 		logger := logger.With(fields.PeerID(pid))
 		raw, err := n.streamCtrl.Request(logger, pid, protocol, encoded)
 		if err != nil {
-			if err != multistream.ErrNotSupported {
+			// TODO: is this how to check for ErrNotSupported?
```

Comment:
Can we have a test for this to be sure?

Reply:
> Can we have a test for this to be sure?

I should definitely verify that this would work, however I could not find the exact lines at which this error is returned from :\

Reply:
Because this condition only affects logging, it's not critical, I'll research this after merging. 

https://github.com/bloxapp/ssv/issues/1013